### ISSUE TITLE
Add # Species in Gazetteer sites*

### ISSUE THREAD

**Issue by Louis-Backstrom** [bird-team/brisbane-bird-atlas#95]
Currently in the gazetteer, I've got it such that each site has information about the name, eBird link, number of species recorded, location and google link, alongside text about the site and some "local experts" (basically just the people with lots of eBird checklists/species).

The problem I can see with the number of species value is twofold - 1. This number isn't currently linked to anything, so it has to be manually updated. This isn't necessarily a problem with some of the more established sites / when there's only 15 sites on the list, but once we've added more, especially those with quickly incrementing species list (ie those out west), this could get painful to do. 2. This number is based directly off live eBird data, not the 3-monthly dataset like the rest of the atlas, which means it's "ahead" - not necessarily a super big issue, but it'd be nice to have some consistency.

My thought is that it should be possible to garner information about sites, as well as species, from the dataset. Each site has a unique identifier (L######) so I'm hoping we could set it up so that the site pulls information from the dataset regarding number of species (and maybe number of checklists too), displays that alongside the rest of the text for that site, and then ranks them according to whatever system - at the moment it's based on number of species seen, but we could change to alphabetical (not a great idea imo) or number of checklists or such. That way it would be "live updating" just like the rest of the atlas. @jeffreyhanson does this sound achievable? It's not a major problem at the moment - focus should be on the species accounts.

**Comment by jeffreyhanson** [comment]
Yeah, it's achievable - it might take me a few hours to get something working though. This would essentially require (i) writing code to calculate the numbers and store it in a new assets zip file, (ii) updating the asset management code to upload and download the the new asset file, (iii) actually making the new asset files and pushing them to GitHub, (iv) writing user-friendly R code to insert the species'  counts into the text in the atlas (e.g. a function like `number_species_in_hotspot("INSERT_ID_HERE")`, (v) replacing the numbers in the gazetteer with these functions, and (vi) pushing the updated gazetter page to GitHub. Also, I might have to add in some extra parameters too. So, if this isn't urgent, I'll see if I can find sometime over the next few weeks and see how I go? How does that sound?

**Comment by Louis-Backstrom** [comment]
Sounds great to me - if Boss Man @dbl3raf gives it the go ahead?

**Comment by dbl3raf** [comment]
I agree it would be great. A couple of considerations make me wonder if we should indeed do this right now though: (i) I can see the Gazetteer / Directory evolving dramatically over time, such that we might in future want a whole heap of calculated data to go in the site entries, such that they end up almost like our species accounts are now, with a data table, several graphs etc, and (ii) @jeffreyhanson time might best be used for automating the creation of survey sheets, and also the general build failure correction and general troubleshooting that he's currently doing. Any thoughts @jeffreyhanson on whether automating the building of survey sheets is possible? I'll open / reopen another issue on that topic, as I know we've discussed it before, and @Louis-Backstrom and I have now come up with a reasonably finalised design for the sheets.

**Comment by dbl3raf** [comment]
I think we should park this issue for now - I have created a "hibernated" label
